### DISCUSSION TITLE
Conventions around docstrings?

### DISCUSSION THREAD

**Discussion by ornamentist** [coalton-lang/coalton#554] [Help!] [answered]
Should Coalton docstrings follow commonly used Common Lisp conventions? For example, UPPERCASE for parameter names and `backtics` for types?

**Comment by eliaslfox** [comment, marked as answer]
We try not to in the standard library. Coalton's stdlib docs get fed into a markdown generator, that only shows type signatures. See the docs [here](https://coalton-lang.github.io/reference/#coalton-library/vector-package). Because parameter  names aren't visible in type signatures we try not to reference them in docstrings.

We do use backtics for types because those look pretty in markdown.

**Reply by ornamentist** [reply]
Do the `;;`, `;;;` conventions for comments also carry across from CL?

**Reply by stylewarning** [reply]
> Do the `;;`, `;;;` conventions for comments also carry across from CL?

I say yes.
